schema/revert/roles/sample-editor/grants.sql

```diff
@@ -0,0 +1,24 @@
+-- Revert seattleflu/schema:roles/sample-editor/grants from pg
+
+begin;
+
+revoke select, insert, update
+    on warehouse.sample
+  from "sample-editor";
```

Comment:
Typically we go a bit broader with the revert script, see this revert template: https://github.com/seattleflu/id3c/blob/master/schema/templates/revert/role/grants.tmpl . (Note that the template also can get used when you create the role with https://github.com/seattleflu/id3c/blob/master/dev/add-role )

Reply:
Ok, that makes sense. I didn't realize there was a template, I was just following `role/manifest-uploader` as an example.